docs/API Reference/cli.md

```diff
@@ -0,0 +1,11 @@
+
+# quilt3 CLI
+Quilt CLI
+
+## `quilt3 catalog`
+Run the Quilt catalog on your machine (**requires Docker**). Running `quilt3 catalog` launches a webserver on your local machine using Docker and a Python microservice that supplies temporary AWS credentials to the catalog. Temporary credentials are derived from your default AWS credentials (or active `AWS_PROFILE`) using `boto3.sts.get_session_token`. For more details about configuring and using AWS credentials in `boto3`, see: [AWS credentials for boto3](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html)
```

Comment:
please edit to 80 character-wide lines otherwise this becomes a pain to edit in vi, etc. newline becomes space for long lines